docs/man_pages/project/testing/run-android.md

```diff
@@ -10,19 +10,20 @@ Start an emulator and run the app inside it | `$ tns run android --emulator [<Em
 Runs your project on a connected Android device or in a native Android emulator, if configured. This is shorthand for prepare, build and deploy. While your app is running, prints the output from the application in the console and watches for changes in your code. Once a change is detected, it synchronizes the change with all selected devices and restarts/refreshes the application.
 
 ### Options
+* `--available-devices` - Shows a list of available emulators to be started and a list of already connected devices.
+* `--device` - Specifies a connected device/emulator to start and run the app.
+* `--emulator` - If set, runs the app in all available and configured android emulators. It will start an emulator if none are already running.
+* `--justlaunch` - If set, does not print the application output in the console.
```

Comment:
>besides that, this works for both emulator on device right?

this? 
* the `--device` flag works with emulators/simulators/real devices
* the `--emulator` flag works only with emulators/simulators
`--justlaunch` doesn't start the application, it only cuts off the console, once the application is running.